src/js/VirtualFC.js

```diff
@@ -19,7 +19,12 @@ const VirtualFC = {
         virtualFC.MIXER_CONFIG.mixer = 3;
 
         virtualFC.MOTOR_DATA = Array.from({length: 8});
-        virtualFC.MOTOR_3D_CONFIG = true;
+        virtualFC.MOTOR_3D_CONFIG = {
+            deadband3d_low: 1406,
```

Comment:
Not sure this is needed for this fix?

Reply:
Yeah maybe make a new issue for it, found another problem (see attached issue)